libs/execution/test/test-logger.ts

```diff
@@ -0,0 +1,187 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import * as chalk from 'chalk';
+import { LangiumDocument } from 'langium';
+import { assertUnreachable } from 'langium/lib/utils/errors';
+import { Range } from 'vscode-languageserver';
+import { uinteger } from 'vscode-languageserver-types';
+
+import { DiagnosticSeverity, Logger } from '../src/lib';
+
+export class TestLogger extends Logger {
```

Comment:
Seems to be the same was the `default-logger`? Can we simplify somehow? Or did I overlook some testing-specific functionality?

Reply:
Yeah that is what we discussed briefly:
I would like to use the `default-logger`, but that is located inside the `interpreter-lib` and I cannot import that due to cyclic dependency.

I would have to refactor the logging and move the `default-logger` into the `execution` lib and I did not want to do this as part of this PR.

Reply:
Right, I remember, Then let's spawn an issue here for it.